### PR TITLE
Switch to C#

### PR DESCRIPTION
**Pull request by naoei**
:sunglasses:

This is a default Rider .NET Core template for the website using Razor.

### CODE REVIEW

File: GDEdit Web/GDE.Web/Data/WeatherForecast.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace GDE.Web.Data
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
+        public int TemperatureC { get; set; }
+
+        public int TemperatureF => 32 + (int) (TemperatureC / 0.5556);
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Data/WeatherForecast.cs]: `public int TemperatureF => 32 + (int)(9 / 5 * TemperatureC);`

File: GDEdit Web/GDE.Web/Pages/Counter.razor [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@page "/counter"
+
+<h1>Counter</h1>
+
+<p>Current count: @currentCount</p>
+
+<button class="btn btn-primary" @onclick="@IncrementCount">Click me</button>
+
+@code {
```
Comment (Rekkonnect) [inline GDEdit Web/GDE.Web/Pages/Counter.razor]: Can we customize where the fuck the brackets are being placed?

**Review by Rekkonnect — CHANGES_REQUESTED** [review]
Interesting, some things were overlooked, maybe this needs a bit of polishing

File: GDEdit Web/GDE.Web/Pages/Counter.razor (hunk 2 above)
Comment (naoei) [reply GDEdit Web/GDE.Web/Pages/Counter.razor]: yes we can.

**Review by Rekkonnect — APPROVED** [review]

**Pull request merged**